keyboards/hhkb/yang/MEMO.md

```diff
@@ -0,0 +1,135 @@
+## Hardware Information
```

Comment:
Could this file be renamed, so it's lower case?

Reply:
Sure